joplin/static/js/admin.js

```diff
@@ -0,0 +1,68 @@
+// TODO: This a better way
```

Comment:
I don't think this file is supposed to be here since we refactored to have static just include the bundles